_drgn.pyi

```diff
@@ -192,8 +192,12 @@ class Program:
     # address_or_name is positional-only.
     def symbol(self, address_or_name: Union[IntegerLike, str]) -> Symbol:
         """
-        Get the symbol containing the given address, or the global symbol with
-        the given name.
+        Get the symbol containing the given address, or symbol with the given
+        name.
+
+        When performing name lookups, global (i.e.
+        :class:`SymbolBinding.GLOBAL`) symbols are preferred, but a local
+        symbol may be returned if there is a local match and no global.
```

Comment:
The mention of `SymbolBinding` made me realize that we probably want to flesh out the rules for the other kinds of bindings. I think we want:

1. If a `GLOBAL` or `UNIQUE` symbol is found, return that. (From what I can tell, `UNIQUE` symbols are just a special case of `GLOBAL`, so we can treat them the same for name lookups.)
2. If a `WEAK` symbol is found, return that. (So `GLOBAL` symbols override `WEAK` symbols, just like when linking.)
3. Return any other matching symbol.